.travis.yml

```diff
@@ -243,7 +243,9 @@ matrix:
           packages:
             - i18nspector
       env:
-        - CHECKS=1
+        - CHECKS=i18nspector
+    - env:
```

Comment:
Apparently this ends up using addons.apt.packages from the very beginning of the file, but [fails](https://travis-ci.org/newsboat/newsboat/jobs/479540667) because `ubuntu-toolchain-r-test` source is not added.

Since that source is used by every other Ubuntu configuration, I suggest adding it to the addons list at the top, and remove the source from all other configurations. That should fix the build, and shorten Travis.yml to boot.

Reply:
I tried to do that, lets see if it works :slightly_smiling_face:

Reply:
Hmm I guess it did not work: https://travis-ci.org/newsboat/newsboat/builds/480144620?utm_source=github_status&utm_medium=notification.
I tried another solution: https://travis-ci.org/newsboat/newsboat/builds/480146950?utm_source=github_status&utm_medium=notification